cli/internal/core/engine_persistent_deps_test.go

```diff
@@ -0,0 +1,233 @@
+package core
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	testifyAssert "github.com/stretchr/testify/assert"
+	"github.com/vercel/turbo/cli/internal/fs"
+	"github.com/vercel/turbo/cli/internal/graph"
+	"github.com/vercel/turbo/cli/internal/util"
+
+	"github.com/pyr-sh/dag"
+)
+
+var WorkspaceGraphDefinition = map[string][]string{
```

Comment:
https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_
```suggestion
var _workspaceGraphDefinition = map[string][]string{
```